flocker/acceptance/endtoend/test_dockerplugin.py

```diff
@@ -93,3 +108,65 @@ def test_run_container_with_volume(self, cluster):
         d.addCallback(lambda _: assert_http_server(
             self, node.public_address, http_port, expected_response=data))
         return d
+
+    def _test_move(self, cluster, origin_node, destination_node):
+        """
+        Assert that Docker can run a container with a volume provisioned by
+        Flocker, shut down the container and then start a new container
+        with the same volume on the specified node.
+
+        :param cluster: The ``Cluster`` to talk to.
+        :param Node origin_node: Original node to start container on.
+        :param Node destination_node: Original node to start container on.
+
+        :return: ``Deferred`` that fires on assertion success, or failure.
+        """
+        data = "hello world"
```

Comment:
:question: I know the volume already has a `random_name`, but I'd be inclined to make this `random_name` too.